src/components/PermissionsManagement/PermissionListItem.jsx

```diff
@@ -167,7 +169,10 @@ function PermissionListItem(props) {
               }}
               disabled={
                 // eslint-disable-next-line react/destructuring-assignment
-                !props.hasPermission('putRole') || immutablePermissions.includes(permission)
+                !(props.isEditing
+                  ? props.hasPermission('putRole')
+                  : props.hasPermission('putRole') || props.hasPermission('postRole')) ||
```

Comment:
```suggestion
                  : props.hasPermission('postRole')) ||
```
Also need to add this check for the category button above.

Reply:
Got it. Also noticed a recent change leading to an error:

<img width="2020" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/96925606/f0f9d51b-497a-4796-95cc-ac65180442cd">
<img width="660" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/96925606/62ce3b9c-a164-42f4-b61b-c038242070d8">